spring-integration-hazelcast/build.gradle

```diff
@@ -41,7 +41,7 @@ ext {
 	hazelcastVersion = '3.5'
```

Comment:
I see the fresh Hazelcast 3.6 version!

Plus we can upgrade to Gradle 2.5, but yes, in the separate commit.
Or I can address that on merge.